cmd/genschema/main.go

```diff
@@ -51,6 +52,10 @@ func main() {
 						corev1.FSGroupChangeAlways,
 					},
 				}
+			case monitoringv1.Duration:
+				return &jsonschema.Schema{
+					Type: "string",
```

Comment:
Can this also include a `Pattern` for duration strings? You can probably pull a good regex from the docs on `monitoringv1.Duration`.

Reply:
ahh ok, i was wondering how deep we should go with this.